### PR TITLE
Added Mammoth Networks to Hosting

### PR DESCRIPTION
**Pull request by trogau**

### CODE REVIEW

**Comment by trogau** [comment]
Argh, whitespace typo.

**Comment by trogau** [comment]
Reopening with a fix for the whitespace typo in hosting.yml.

**Comment by jamcat22** [comment]
Great. Are there any better doc urls? Other than this post all I can see is one behind a login page.

File: _data/hosting.yml [hunk 1]
```diff
@@ -71,6 +71,13 @@ websites:
       twitter: iPage
       img: ipage.png
       tfa: No
+      
+    - name: Mammoth Networks
+      url: https://www.mammoth.net.au/
+      img: mammothnetworks.png
+      tfa: Yes
+      software: yes
```
Comment (Carlgo11) [inline _data/hosting.yml]: Change to `Yes` please.

Comment (trogau) [inline _data/hosting.yml]: Fixed the 'Yes' thing (sorry!)

Re: the doc link, there's nothing with any more information that's obvious.
If there's anything specific that you like seeing on Doc links I should be
able to get it added!

## 

dave

**Pull request merged**